<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>This file is part of the xTuple ERP: PostBooks Edition, a free and
open source Enterprise Resource Planning software suite,
Copyright (c) 1999-2012 by OpenMFG LLC, d/b/a xTuple.
It is licensed to you under the Common Public Attribution License
version 1.0, the full text of which (including xTuple-specific Exhibits)
is available at www.xtuple.com/CPAL.  By using this software, you agree
to be bound by its terms.</comment>
 <class>configureSearchPath</class>
 <widget class="QWidget" name="configureSearchPath">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set Search Path</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="XLabel" name="_excludedLit">
     <property name="text">
      <string>Schemas Not in Order:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="XLabel" name="_includedLit">
     <property name="text">
      <string>Schemas in Order:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="XTreeWidget" name="_excluded">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>4</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <layout class="QVBoxLayout" name="_addButtonLayout">
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>119</width>
         <height>47</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="_add">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Add -&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_addAll">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Add All -&gt;&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="_remove">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&lt;-Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_removeAll">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&lt;&lt;-Remove All</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>110</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="3">
    <widget class="XTreeWidget" name="_included">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>4</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <layout class="QVBoxLayout" name="_moveButtonLayout">
     <item>
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="_moveUp">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Move Up</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_moveDown">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Move Down</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="XCheckBox" name="_showAll">
     <property name="text">
      <string>Show all schemas</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="3">
    <widget class="QPlainTextEdit" name="_result">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2" colspan="3">
    <widget class="XLabel" name="_resultLit">
     <property name="text">
      <string>New Search Path:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XCheckBox</class>
   <extends>QCheckBox</extends>
   <header>xcheckbox.h</header>
  </customwidget>
  <customwidget>
   <class>XLabel</class>
   <extends>QLabel</extends>
   <header>xlabel.h</header>
  </customwidget>
  <customwidget>
   <class>XTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>xtreewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_excluded</tabstop>
  <tabstop>_add</tabstop>
  <tabstop>_addAll</tabstop>
  <tabstop>_remove</tabstop>
  <tabstop>_removeAll</tabstop>
  <tabstop>_included</tabstop>
  <tabstop>_moveUp</tabstop>
  <tabstop>_moveDown</tabstop>
  <tabstop>_showAll</tabstop>
  <tabstop>_result</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
